examples/rainfall/rainfall/server.py

```diff
@@ -19,7 +19,13 @@ def cell_portrayal(cell: LakeCell) -> Tuple[float, float, float, float]:
     if cell.water_level == 0:
         return cell.elevation, cell.elevation, cell.elevation, 1
     else:
-        return 0, 0, 255, 1
+        # return scaled blue color for water level
+        return (
```

Comment:
Is this supposed to be a color gradient? I couldn't spot any noticeable gradient. What if you exaggerate the gradient further?

Reply:
The screenshot was taken after 20 steps. Is it slightly better after 100 steps (or not)?

<img width="435" alt="Screen Shot 2022-10-09 at 8 53 59 PM" src="https://user-images.githubusercontent.com/10785873/194787417-f58bb1e4-2077-47f1-af4a-34976881f7e2.png">

I guess initially there may not be enough raindrops to make a difference.

Reply:
I see. You probably need to add a comment explaining that an array returns a color gradient.

Reply:
Something like the recent update?

Reply:
Not enough info. People are not likely to be aware that an array instead of a scalar means gradient.

Reply:
Hmm how about now

Reply:
Huh it does mean a single color after all. I thought we have color varying across space, and within a cell. In https://github.com/projectmesa/mesa/blob/fed83543487f6834b172f3a7c88754733e12c539/examples/shape_example/shape_example/server.py#L18, if you specify 2 color in an array, it becomes a gradient of color within 1 portrayal.

Reply:
Looks better, merging.

Reply:
Yes sorry about the confusion 😅